<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="jenkins-agent-10-25-195-249" failures="0" tests="6" name="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest" time="60.333" errors="0" timestamp="2023-09-05T07:00:37 UTC" skipped="0">
  <testcase classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest" name="invokeAsyncTestWith_503" time="40.046"/>
  <system-out/>
  <testcase classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest" name="invokeAsyncTestWith_504_And_No_MaxRetryTime" time="0.048"/>
  <system-out/>
  <testcase classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest" name="invokeAsyncTestWith_429" time="5.062"/>
  <system-out/>
  <testcase classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest" name="invokeAsyncTestWith_429_And_No_MaxRetryTime" time="0.051"/>
  <system-out/>
  <testcase classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest" name="invokeAsyncTestWith_502" time="13.061"/>
  <system-out/>
  <testcase classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest" name="invokeAsyncTestWith_504" time="2.065"/>
  <system-out/>
</testsuite> <!-- com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest -->
